GameServer.cpp

```diff
@@ -627,14 +623,12 @@ void GameServer::SendGameSessionRequest(sockaddr_in &to, unsigned int serverRand
 			// Check if we got the right version
 			if (wsaData.wVersion != version)
 			{
-				// Wrong version
 				WSACleanup();
-				return WinsockVersionFailed;	// Error: Wrong Winsock version
+				throw std::runtime_error("Incorrect version of Winsock initialized in GameServer");

```

Comment:
We might want to include the expected and actual Winsock versions in the error message.